moped-editor/src/auth/user.js

```diff
@@ -246,23 +249,21 @@ export const UserProvider = ({ children }) => {
 
       getCognitoSession()
         .then(async (session) => {
+          // Initialize user data on app reload/session resume before proceeding to navigation.
+          // Some queries rely on user database data like user id for followed projects.
+          // We must populate userDatabaseData if it's null otherwise the query will fail
+          // when users are redirected to their last route after a forced logout
+          // (see MainLayout.js and DashboardLayout.js for previous route restoration handling).
           if (session) {
-            // Initialize user data on app reload/session resume before proceeding to navigation.
-            // Some queries rely on user database data like user id for followed projects.
-            // We must populate userDatabaseData if it's null otherwise the query will fail
-            // when users are redirected to their last route after a forced logout
-            // (see MainLayout.js and DashboardLayout.js for previous route restoration handling).
             const userDBData = await initializeUserDBObject(session);
             setSessionDatabaseData(userDBData);
-          }
-
-          setUser(session);
-          setIsLoginLoading(false);
 
-          if (session === null) {
+            setUser(session);
+          } else {
             destroyProfileColor();
             deleteSessionDatabaseData();
           }
+          setIsLoginLoading(false);
```

Comment:
Need to turn off the loading spinner whether there is a session or not. Otherwise, we get stuck at the login screen with the spinner blocking any interactions to log back in.

thanks @tillyw for catching this!! 🙏